k8s/agones/Makefile

```diff
@@ -0,0 +1,112 @@
+include ../app.Makefile
+include ../crd.Makefile
+include ../gcloud.Makefile
+include ../var.Makefile
+
+TAG ?= 0.8.1
+DEMO_RELEASE_TAG ?= release-0.8.0
+NAME ?= agones
+
+
+$(info ---- TAG = $(TAG))
+$(info ---- DEMO_RELEASE_TAG = $(DEMO_RELEASE_TAG))
+
+
+# source repo and destination tags for images
+SOURCE_REGISTRY ?= gcr.io/agones-images
+APP_DEPLOYER_IMAGE ?= $(REGISTRY)/agones/deployer:$(TAG)
+AGONES_CONTROLLER_IMAGE ?= $(REGISTRY)/agones:$(TAG)
+AGONES_SDK_IMAGE ?= $(REGISTRY)/agones/agones-sdk:$(TAG)
+AGONES_PING_IMAGE ?= $(REGISTRY)/agones/agones-ping:$(TAG)
+
+
+XONOTIC_DEMO_BASE=https://github.com/GoogleCloudPlatform/agones/raw/$(DEMO_RELEASE_TAG)/examples/xonotic
+
+
+APP_PARAMETERS ?= { \
+    "APP_INSTANCE_NAME": "$(NAME)", \
+    "NAMESPACE": "$(NAMESPACE)" \
+}
+
+
+APP_TEST_PARAMETERS ?= { \
+}
+
+
+app/build:: .build/agones/helmdeps \
+            .build/agones/xonotic.yaml \
+            .build/agones/deployer \
+            .build/agones/agones-controller \
+            .build/agones/agones-sdk \
+            .build/agones/agones-ping
+
+
+.build/agones: | .build
+	mkdir -p "$@"
+
+
+.build/agones/helmdeps: chart/agones-mp/requirements.* \
```

Comment:
I think it's OK to submit this as it is for now. However, as follow-up, we should keep the copy of the helm chart in the click-to-deploy repo. We can move this step to the `make app/upstream` and run it, every time upstream is needed. 

This will give us more control on how the chart was changed, and it's looks like.

Please let me know, if you agree, please create a TODO comment to track this.

Reply:
I'm committing the Helm chart as part of the git commit, and I've created a app/upstream-chart target to update the chart if required.

Reply:
Done